run.py

```diff
@@ -0,0 +1,12 @@
+import uvicorn
+
+
+if __name__ == "__main__":
+    uvicorn.run(
+        "app.server:APP",
+        host="0.0.0.0",
+        port=9096,
+        reload=True,
+        reload_dirs=["app"],
+        log_config=None,
+    )
```

Comment:
Perhaps we should snag some these inputs from a CLI and specify workers?

Reply:
I'm not sure this is necessary. When the ARS goes to deploy, they'll be using helm charts where the port will be specified and number of workers. This run.py file was just for my development (with automatic reloads on code change), and when the docker image is used in deployment, the main.sh file is the one that's uses which allows a port to be specified.